ra/ra.go

```diff
@@ -634,10 +634,24 @@ func (ra *RegistrationAuthorityImpl) NewAuthorization(ctx context.Context, reque
 				identifier.Value,
 				err)
 		} else if err == nil {
+			fmt.Println("uhjh?")
 			return *pendingAuth, nil
 		}
 	}
 
+	if features.Enabled(features.V1DisableNewValidations) {
+		exists, err := ra.SA.PreviousCertificateExists(ctx, &sapb.PreviousCertificateExistsRequest{
+			Domain: &identifier.Value,
+			RegID:  &regID,
+		})
+		if err != nil {
+			return core.Authorization{}, err
+		}
+		if !*exists.Exists {
+			return core.Authorization{}, berrors.UnauthorizedError("Validations for new domains are disabled in the V1 API")
```

Comment:
Can you add a link to [the community forum ACME v1 shutdown post](https://community.letsencrypt.org/t/end-of-life-plan-for-acmev1/88430) in this error message? It'll be a long ugly message but I think it's valuable in making 100% crystal clear that it isn't a bug or something but an intentional turn-down.